### PR TITLE
Fixing spelling

### PR DESCRIPTION
**Pull request by Thienanbad**
Fixing some spelling

### CODE REVIEW

File: jsons/Beliefs.json [hunk 1]
```diff
@@ -433,7 +433,7 @@
         "name": "Tithe",
         "type": "Founder",
         "uniques": [
-            "[+1 Gold] for every [4] global followers [in all cities]"
+            "[+1 Gold] form every [4] global followers [in all cities]"
```
Comment (yairm210) [inline jsons/Beliefs.json:436]:
```suggestion
            "[+1 Gold] from every [4] global followers [in all cities]"
```

File: jsons/Beliefs.json [hunk 2]
```diff
@@ -483,7 +483,7 @@
         "name": "World Church",
         "type": "Founder",
         "uniques": [
-            "[+1 Culture] for every [2] global followers [in foreign cities]"
+            "[+1 Culture] form every [2] global followers [in foreign cities]"
```
Comment (yairm210) [inline jsons/Beliefs.json:486]:
```suggestion
            "[+1 Culture] from every [2] global followers [in foreign cities]"
```
You were so close!
Good catches!

**Comment by Thienanbad** [comment]
Oke can you check it 1 more time and merge it? @yairm210

**Comment by yairm210** [comment]
You didn't actually solve the second one... You just marked it as resolved...

**Comment by Thienanbad** [comment]
> You didn't actually solve the second one... You just marked it as resolved...

Oke i have fixed it

**Pull request merged**